src/js/models/geocoder/GoogleMapsAutocompleter.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+define(
+  ['backbone', 'gmaps', 'models/geocoder/Prediction'],
+  (Backbone, gmaps, Prediction) => {
+    /**
+    * @class GoogleMapsAutocompleter
+    * @classdes Integrate with the Google Maps Places Autocomplete API using the
+    * Google Maps AutocompleteService JS library.
+    * @classcategory Models
+    */
+    const GoogleMapsAutocompleter = Backbone.Model.extend({
```

Comment:
Care to update this and Geocoder to use ES6 class as well?

Reply:
Done